integrations/mistral.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: integration
+name: Mistral
+description: This page demonstrates how to use OpenAIGenerator within Haystack to make use of Mistral models.
+authors:
+    - name: deepset
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: deepset-ai
+pypi: https://pypi.org/project/haystack-ai
+repo: https://github.com/deepset-ai/haystack
+type: Model Provider
+report_issue: report_issue: https://github.com/deepset-ai/haystack/issues
+logo: /logos/mistral.png
+version: Haystack 2.0
+toc: true
+---
+
+This page demonstrates how to use the [OpenAIChatGenerator](https://docs.haystack.deepset.ai/v2.0/docs/openaichatgenerator) and [OpenAITextEmbedder](https://docs.haystack.deepset.ai/v2.0/docs/openaitextembedder) within Haystack to make use of Mistral models.
+
```

Comment:
Can we add a single line here that explains what's going on here something like:
"We're able to use the OpenAI generators for Mistral as they use the same protocol".. And then add a line explaining that this is possible by changing the api_base_url